src/base/static/components/organisms/left-sidebar.js

```diff
@@ -24,9 +23,6 @@ const LeftSidebar = props => {
       />
       {props.leftSidebarComponent === "MapLayerPanel" && <MapLayerPanel />}
       {props.leftSidebarComponent === "MapFilterPanel" && <MapFilterPanel />}
-      {props.leftSidebarComponent === "MapLegendPanel" && (
-        <MapLegendPanel config={props.mapLegendPanelConfig} />
-      )}
```

Comment:
I dropped support for including MapLegendPanel in the left sidebar. It was only used on the NYRP flavors.